.github/workflows/main_ci.yml

```diff
@@ -24,10 +24,21 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install --upgrade numpy mypy
+          python -m pip install --upgrade numpy mypy cmake pytest pybind11
+      - name: Install pykokkos-base
+        run: |
+          cd /tmp
+          git clone https://github.com/kokkos/pykokkos-base.git
+          cd pykokkos-base
+          mkdir build && cd build
+          cmake -DENABLE_LAYOUTS=ON -DENABLE_MEMORY_TRAITS=OFF ../
+          make -j
       - name: Install pykokkos
         run: |
           python -m pip install .
       - name: mypy check
         run: |
           mypy pykokkos
+      - name: run tests
+        run: |
+          python runtests.py
```

Comment:
The fact that the GitHub Actions jobs "pass" likely reveals an issue with `runtests.py`--I suspect it is not properly returning a non-zero exit code when `pytest` fails, so we should probably fix that--either while working on this PR or separately, since it would be important to see the CI fail when tests fail of course.

Reply:
Ok, I pushed in a revision to `runtests.py` to address this--looks like the exit code is properly propagated to a CI failure caused by the testsuite now.

Reply:
looks good, thanks!